.github/workflows/pr-checks.yml

```
@@ -16,6 +16,17 @@ jobs:
         run: |
           ! grep 'uses: github/codeql-variant-analysis-action/' .github/workflows/codeql-query.yml | grep -q -v '@main$'
 
+  integration-tests-have-been-run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check label is present on PR
+        if: ${{ "$GITHUB_EVENT_NAME" == 'pull_request' }}
```

Minor: If this works, then I have no problem with it. I didn't know that we could use environment variables in `if` expressions.

```suggestion
        if: ${{ github.event_name == 'pull_request' }}
```

---

Good point. I'd forgotten this was a special actions `if` expression.